communications/events/messages.py

```diff
@@ -4,32 +4,34 @@
 from channels.db import database_sync_to_async
 from channels.layers import get_channel_layer
 
-from communications.entities.messages import Message
+from communications.domain.entities.messages import Message, ChatRoom
 from communications.events.base import BaseEvent
 
 logger = logging.getLogger(__name__)
 
 
 @dataclass
 class MessageNotificationEvent(BaseEvent):
-    async def trigger(self, message: Message):
+    def trigger(self, message: Message, chat: ChatRoom):
         from notifications.models import Notification  
         try:
-            await database_sync_to_async(Notification.objects.create)(
-                investor_id=message.sender_id,
-                startup_id=message.receiver_id,
+            Notification.objects.create(
+                investor_id=chat.sender_id,
+                startup_id=chat.receiver_id,
+                notification_type=2,
+                message_id=message.oid
             )
 
-            logger.info(f"Notification object created for message from sender {message.sender_id}")
+            logger.info(f"Notification object created for message from sender {chat.receiver_id}")
 
-            channel_layer = get_channel_layer()
-            await channel_layer.group_send(
-                f'notifications_{message.receiver_id}',
-                {
-                    'type': 'send_notification',
-                    'notification': f'New message: {message.content}',
-                }
-            )
+            # channel_layer = get_channel_layer()
+            # channel_layer.group_send(
+            #     f'notifications_{chat.receiver_id}',
+            #     {
+            #         'type': 'send_notification',
+            #         'notification': f'New message: {message.content.as_generic_type()}',
+            #     }
+            # )
```

Comment:
commented lines of code don't look like a good solution on a real project, although for a training project it's acceptable

Reply:
i know